packages/help-callout/src/HelpCallout.tsx

```diff
@@ -0,0 +1,33 @@
+import { Box } from '@ag.ds-next/react/box';
+import { Callout } from '@ag.ds-next/react/callout';
+import { CoreProvider } from '@ag.ds-next/react/core';
+import { TextLink, TextLinkExternal } from '@ag.ds-next/react/text-link';
+
+interface HelpCalloutProps {
+	minimal?: boolean;
+}
+
+export const HelpCallout = (props: HelpCalloutProps) => (
+	<CoreProvider>
+		<Callout title="Need more help?">
+			{props.minimal === true ? null : (
+				<Box>
+					Access our{' '}
+					<TextLinkExternal href="/help">help articles</TextLinkExternal>
```

Comment:
@jordanoverbye think this should be hardcoded to the full domain name?

this works for dev environments but means this help callout won't work outside of export service.

maybe we could accept domain name as an optional prop?

Reply:
Maybe we should just make this a `href` prop, and default to `/help`?

Reply:
`minimal` seems like a bit of an ambiguous name, maybe `showHelpArticles` would be better? And default to `true`.